packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummary.js

```diff
@@ -59,11 +54,21 @@ const PriceSummary = props => {
 
     const { subtotal, total, discounts, giftCards, taxes, shipping } = flatData;
 
-    const priceClass = isCartUpdating ? classes.priceUpdating : classes.price;
-    const totalPriceClass = isCartUpdating
+    const priceClass = isUpdating ? classes.priceUpdating : classes.price;
+    const totalPriceClass = isUpdating
         ? classes.priceUpdating
         : classes.totalPrice;
 
+    const proceedToCheckoutButton = !isCheckout ? (
+        <div className={classes.checkoutButton_container}>
+            <Button disabled={isUpdating} priority={'high'}>
+                <Link to={'/checkout'} className={classes.images}>
+                    {'Proceed to Checkout'}
+                </Link>
+            </Button>
```

Comment:
I think if people wanted to they could extend the `Link` to trigger some analytic handler. Certainly something to be aware of.

Reply:
This is invalid HTML. [You can't put an `<a>` element inside a `<button>`.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)

> **Permitted content:** Phrasing content but there must be no Interactive content

If this is a submit button for a form, handle the navigation in the form's `onSubmit` handler; otherwise, either handle the navigation in the button's `onClick` handler or style the `Link` to match the desired appearance.